shared/templates/mount_option_home/ansible.template

```diff
@@ -0,0 +1,64 @@
+# platform = multi_platform_all
+# reboot = false
+# strategy = configure
+# complexity = low
+# disruption = high
+
+-   name: "Initialize variables"
```

Comment:
Please add `{{{ rule_title }}}: ` to each Ansible Task name so that the same Ansible Tasks can be distinguished between rules.

Reply:
Thank you for the review, I updated this